bento-platform/bentoV2
# Tag branch union

### Changes
- Added a new env file which overshadows the .env file and is much smaller for easier dev.
- Readme includes changes on how to use this. As a way to test the readme, read the readme to test this PR.
- Combined environment variables for tags and branches for containers
- Added the new vars to dev and deploy templates

### To test 
- change a variable for tag/branch for a container in local.env
- clean the container
- run the container
- check the branch of the container in container logs
